app/build.gradle

```diff
@@ -23,4 +23,6 @@ dependencies {
     implementation 'com.android.support:support-compat:28.0.0-rc01'
     implementation 'com.android.support:appcompat-v7:28.0.0-rc01'
     implementation 'com.android.support:design:28.0.0-rc01'
+
+    testCompile 'junit:junit:4.12'
```

Comment:
Change it to `testImplementation` because `Compile` is deprecated.

Reply:
👍 fixing that, thx.